t/conn.t

```diff
@@ -235,3 +235,52 @@ committed: true
 --- no_error_log
 [error]
 [lua]
+
+
+=== TEST 5: a single key (set_conn && set_burst)
+--- http_config eval
+"
+$::HttpConfig
+
+    lua_shared_dict store 1m;
+"
+--- config
+    location = /t {
+        content_by_lua '
```

Comment:
New tests should use the `_by_block` directive instead

Reply:
`content_by_lua` and `content _by_lua_block` are the same, isn't it? And now it works well and correctly .

Reply:
`content_by_lua` and `content _by_lua_block` are the same, isn't it? And now it works well and correctly .

Reply:
`content_by_lua` does not come without some minor drawbacks and its use is discouraged as per the documentation. It is now required across all OpenResty repositories to use the later, `by_lua_block` form for all added tests. You will also notice that all or almost all examples across the documentation have been updated to use the newer form as well.

Reply:
Only the new one :)

Reply:
@pearzl `content_by_lua` requires crazy escaping sequences due to nginx string literal escaping requirements. That's why we want to avoid using it (and its siblings) in newly added test cases ourselves.

Reply:
OK, I understand, it's indeed a better choice.